app/src/main/res/layout/activity_main.xml

```diff
@@ -25,12 +25,12 @@
             android:layout_gravity="bottom|end"
             android:layout_marginTop="550dp"
             android:backgroundTint="?attr/colorPrimary"
-            app:srcCompat="@drawable/ic_baseline_add_24"
+            android:contentDescription="@string/addMeeting"
```

Comment:
Added a content description to the floating action button in the form of a new string value in strings.xml